crates/rpc/src/routes/beacon.rs

```diff
@@ -28,6 +35,18 @@ pub fn get_beacon_routes(
         .and_then(move || get_genesis(network_spec.genesis.clone()))
         .with(log("genesis"));
 
+    let fork = {
+        beacon_base
+            .and(path("states"))
+            .and(param::<ID>())
+            .and(path("fork"))
+            .and(end())
+            .and(get())
+            .and(db_filter.clone())
+            .and_then(move |state_id: ID, db: ReamDB| get_fork(state_id, db))
+            .with(log("fork"))
+    };
```

Comment:
```suggestion
    let fork = beacon_base
            .and(path("states"))
            .and(param::<ID>())
            .and(path("fork"))
            .and(end())
            .and(get())
            .and(db_filter.clone())
            .and_then(move |state_id: ID, db: ReamDB| get_fork(state_id, db))
            .with(log("fork"));
```

now that we are using a filter, could you remove the extra scope for fork and the other endpoints